openandroidinstaller/views/requirements_view.py

```diff
@@ -251,6 +255,30 @@ def get_battery_check(self):
         )
         return battery_checkbox, battery_check_card
 
+    def get_boot_stock_check(self):
+        """Get checkbox and card for default requirements: boot stock once."""
+        boot_stock_checkbox = Checkbox(
+            label="Booted the stock OS at least once.",
+            on_change=self.enable_continue_button,
+        )
+        boot_stock_check_card = Card(
+            Container(
+                content=Column(
+                    [
+                        Markdown(
+                            """
+#### Boot your device with the stock OS at least once and check every functionality.
+Make sure that you can send and receive SMS and place and receive calls (also via WiFi and LTE, if available), otherwise it won\'t work on LineageOS either! Additionally, some devices require that VoLTE/VoWiFi be utilized once on stock to provision IMS.
```

Comment:
Maybe it should be worded like "otherwise it may not work on other custom ROMs either!"(?), to not focus that much on LineageOS...